### PR TITLE
Make monthly donations tenant aware

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

File: bluebottle/recurring_donations/mails.py [hunk 1]
```diff
@@ -19,16 +20,17 @@ def mail_monthly_donation_processed_notification(monthly_order):
 
     subject = _("Thank you for your monthly support")
 
-    translation.activate(cur_language)
+    translation.activate(receiver.primary_language)
```
Comment (aethemba) [inline bluebottle/recurring_donations/mails.py]: Now that we have a default primary language, you might as well remove the lines above this line that checks if there is a primary language.

File: bluebottle/recurring_donations/mails.py [hunk 2]
```diff
@@ -19,16 +20,17 @@ def mail_monthly_donation_processed_notification(monthly_order):
...
 
     send_mail(
-        template_name='monthly_donation.mail',
+        template_name='recurring_donations/mails/monthly_donation.mail',
         subject=subject,
+        site=tenant_url(),
+        to=receiver,
         order=monthly_order,
-        receiver_first_name=receiver.first_name.capitalize(),
+        first_name=receiver.first_name.capitalize(),
```
Comment (aethemba) [inline bluebottle/recurring_donations/mails.py]: Do we need to change the email templates? We'll lose the translation strings and we need to regenerate and re-enter new translations?

Comment (gannetson) [inline bluebottle/recurring_donations/mails.py]: I just moved the templates into a directory to match other mail templates.
They were in root template dir, now in recurring_donations/mail/
I think translations won't be different.

File: bluebottle/recurring_donations/templates/recurring_donations/mails/monthly_donation.mail.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 {% blocktrans %}
-Hi {{ receiver_first_name }},<br/>
+Hi {{ first_name }},<br/>
```
Comment (aethemba) [inline bluebottle/recurring_donations/templates/recurring_donations/mails/monthly_donation.mail.html]: Doesn't that change the translation string?

Comment (aethemba) [inline bluebottle/recurring_donations/templates/recurring_donations/mails/monthly_donation.mail.html]: @gannetson , if you're sure this doesn't change the translation string then I can merge this PR

Comment (aethemba) [inline bluebottle/recurring_donations/templates/recurring_donations/mails/monthly_donation.mail.html]: @gannetson , what about this transstring?

Comment (aethemba) [inline bluebottle/recurring_donations/templates/recurring_donations/mails/monthly_donation.mail.html]: @gannetson , I hardly dare to ask..

Comment (gannetson) [inline bluebottle/recurring_donations/templates/recurring_donations/mails/monthly_donation.mail.html]: @aethemba I'll schedule it in for some time soon

**Pull request merged**